terraform/repositories.tf

```diff
@@ -40,6 +40,9 @@ locals {
       "checks" : local.checks.drone
     },
     "callisto-auth-keycloak" : {
+      "checks" : ["terraform-validate"]
+    },
+    "callisto-devops" : {
```

Comment:
The checks are configured in the GitHub repo for branch protection so it's not related to the drone permissions. It controls the bit of the PR that says you can't merge until these checks are completed. 
I didn't add  local.checks.drone to callisto-devops because I didn't think it contains any pull_request or push pipelines at the moment.